Gruntfile.js

```diff
@@ -1,112 +1,115 @@
+/* globals module, require */
+
+const loadGruntTasks = require('load-grunt-tasks');
```

Comment:
The require method is a little bit more complex. It can be made global by /* globals */ but this it is not possible to have it inside the module.exports so I decided to declare it outside. The eslint rule is "global-require".